TAF/config/global_variables.py

```diff
@@ -43,6 +43,7 @@
     CORE_METADATA_PORT = "8443/core-metadata"
     CORE_COMMAND_PORT = "8443/core-command"
     CORE_KEEPER_PORT = "8443/core-keeper"
+    PROXY_AUTH_PORT = "8443"
```

Comment:
Also add the `PROXY_AUTH_PORT` variable in the `else` condition, or do not set PROXY_AUTH_PORT here, to avoid the error `Error in file '$HOME/TAF/testScenarios/integrationTest/UC_ACL/acl.robot' on line 11: Setting variable '${proxyAuthUrl}' failed: Variable '${PROXY_AUTH_PORT}' not found` when running integration tests in non-secty mode.